src/components/Semester.vue

```diff
@@ -319,21 +322,21 @@ export default {
       const _this = this;
       for (let i = 0; i < splitReq.length; i++) {
         if (splitReq[i].indexOf('"') >= 0) {
-          splitReq[i] = 'true';
-        }
-        if ('()/, '.indexOf(splitReq[i]) < 0) {
+          //don't include things like 'permission of instructor'
+          splitReq[i] = '';
+          //remove operator before & parentheses if directly before and after
+          if(splitReq[i] !== '(') {
+            splitReq[i-1] = '';
+          } else if(splitReq[i+1] === ')') {
+            splitReq[i-1] = '';
+            splitReq[i+1] = '';
```

Comment:
This one also feels dangerous, like it seems like this could potentially lose information about the difference between (A and B) and (A/B) or something similar? Why were these added? I'm sure it's a good reason I just don't understand the format this is coming in as well.

Reply:
General reasoning for this part of the code is in the other comment

Not sure what you mean by the difference between (A and B) and (A/B)?  (A and B) should be written as (A,B) but I don't know what you mean by losing information

Reply:
yep I was being dumb sorry